src/app/modules/faq/index.tsx

```diff
@@ -1,13 +1,73 @@
-import {FAQModuleContainer, FAQModuleBackground} from "@/app/modules/faq/index.styles";
+import {
+  FAQModuleContainer,
+  FAQModuleBackground,
+} from "@/app/modules/faq/index.styles";
 import SpaceGrotesk from "@/app/components/shared/fonts/space-grotesk";
+import Accordion from "@/app/components/pages/Accordion";
+import { useMobileDetect } from "@/app/hooks/useMobileDetect";
+
+const FAQItems = [
```

Comment:
Temporary items, who should I ask for the FAQ questions? Should I just reuse the ones from last year?

Reply:
Marketing will supply us with those, temp is good for now